src/typescript/azure-openapi-validator/rules/PathParameterMustNotBeEmpty.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const PathParametersMustNotBeEmpty: string = "PathParametersMustNotBeEmpty";
+
+rules.push({
+  id: "R2067",
```

Comment:
nit: make sure you're picking a unique rule id

Reply:
yes I searched in the source code it was unique. Also checked in the doc link you provided now and it is not referenced anywhere.